STL_Extension/doc/STL_Extension/STL_Extension.txt

```diff
@@ -393,3 +393,35 @@ void foo()
 */
 } /* namespace CGAL */
 
+/*!
+\page FromBoostNPtoCGALNP Upgrading Code using \boost Parameters to \cgal Named Function Parameters
```

Comment:
As a general remark, I really dislike how this pretty much Mesh_3-specific code coming into the STL_Extension package. It's a pain to navigate when you're looking at the code, you have to jump around all the time.

We have put graph traits & such next to the data structures, not in BGL: I think the NP framework should be in BGL (or its own package) for the core workings, and NPs + NP helpers should be in the package that relates them.

Reply:
I agree. I propose we create an issue from this comment and let this as is while we come up with a good solution. Would it be OK for you?

Reply:
Sure.

I am not sure how to handle NPs (and NPs helpers) that are shared between really different packages, though, as they cannot be placed into multiple packages. Maybe an independent package is the cleanest, and then each package that uses NP has its own `CGAL/Pkg/parameter_interface.h` + `CGAL/Pkg/Named_parameters.h` headers.